docs/pages/header-file.page.server.mdx

```diff
@@ -62,38 +45,57 @@ export default {
 
 Vike throws:
 ```
-[Wrong Usage] The code of config 'Page' cannot live inside /pages/some-page/+config.h.js
+[Wrong Usage] The code of Page cannot live inside +config.h.js
 ```
 
-Do this instead:
+Instead, runtime code always needs to be imported:
 
 ```js
-// /pages/some-page/+config.h.js
+// +config.h.js
 
 import Page from './some-file'
 import onBeforeRender from './some-other-file'
 
 export default { Page, onBeforeRender }
 ```
 
+> Importing runtime code works because <Link href="#transform">Vike transforms `.h.js`</Link> files.
+
+
+## Optional
+
+Instead of `+config.h.js` you can define a `+config.js` file (without `.h`):
+
+```js
+// +config.js
+
+export default {
+  Page: 'import:./some-file:someExport',
+  onBeforeRender: 'import:./some-other-file:someOtherExport'
+}
```

Comment:
Actually I can also do that in a `+config.h.js` (I just tried it out). Is there anything I can only do in a `+config.js` but not in a `+config.h.js`?

Reply:
Ok actually I think I know the answer: in a `+config.js` if I do imports they won't be transformed. So I can import functions and use them but I won't be able to re-export them. It can be useful in some cases. Maybe that's what we should show here? Otherwise if we don't want to advertise the possibility to import without transformation, then maybe we shouldn't advertise `+config.js` at all?

Reply:
I see. I'm realizing the docs should further explain why the transformation is needed (and explain why code extraction is problematic). I tried but I realized, while trying, that it would be quite time consuming to properly explain all that. So I'm thinking to postpone working on this after the V1 design release.

Reply:
And I think you're right: 9a4d978d2576dd05f1d485072641c1b9fde8bc9a. I'm realizing it's contra productiv to open the "can of worms" without properly explaining everything.